pkg/asset/manifests/infrastructure.go

```diff
@@ -82,13 +66,50 @@ func (i *Infrastructure) Generate(dependencies asset.Parents) error {
 		},
 		Status: configv1.InfrastructureStatus{
 			InfrastructureName:   clusterID.InfraID,
-			Platform:             platform,
 			APIServerURL:         getAPIServerURL(installConfig.Config),
 			APIServerInternalURL: getInternalAPIServerURL(installConfig.Config),
 			EtcdDiscoveryDomain:  getEtcdDiscoveryDomain(installConfig.Config),
+			PlatformStatus:       &configv1.PlatformStatus{},
```

Comment:
The MCO was reading `Platform`- is it just enough to switch to `PlatformStatus`? cc @sgreene570

Reply:
This is only set in new 4.2 clusters.

4.1 clusters do not have `.status.platformStatus` set.

Reply:
I know, so it's ok to switch to the new platformStatus in 4.2 and live with that as you're doing in this PR?